### DISCUSSION TITLE
Where does pro.javacard.capfile come from?

### DISCUSSION THREAD

**Discussion by spaikmos** [martinpaljak/GlobalPlatformPro#344] [Q&A]
/library/src/main/java/module-info.java refers to package "pro.javacard.capfile".  Where does this come from?  I *think* this is from the [capfile](https://github.com/martinpaljak/capfile/tree/master) project, but that one only defines package "pro.javacard".

When I change all references in GlobalPlatformPro from pro.javacard.capfile --> pro.javacard, it links correctly to capfile.  But, I'd like to know how I can compile GPPro without having to alter the source code.  AIUI there's no way to change imports in Java, so I think I'm missing something?

**Comment by martinpaljak** [comment]
This has been moved to ant-javacard, as it is used there and for backwards compatibility needs to retain java 8 support, whereas GPPro is targeting Java 11/17

https://github.com/martinpaljak/ant-javacard/tree/master/capfile
